### PR TITLE
Removes spider webs to prevent mass spam.

### PR DESCRIPTION
**Pull request by Cameron653**
Completely eliminates spam of spiderwebs by commenting it out.

Leaves in the spider binding suit thingy to allow spiders to still rp binding people up, but removes them being able to place down all the damn tiles.

Addresses #450

### CODE REVIEW

File: code/modules/vore/spiderTaurPowers.dm [hunk 1]
```diff
@@ -21,20 +21,20 @@ obj/item/clothing/suit/web_bindings
...
-	if(nutrition >= 30) //BUGS!
+	if(nutrition >= 500) //People decided to abuse it. Sorry. It was asked to be made so it couldn't be spammed, and what do ya know, people are spamming it everywhere.
 		src.visible_message("\blue \the [src] weaves a web from their spinneret silk.")
-		nutrition -= 30 //Squash the bugs!
+		nutrition -= 500
 		spawn(30) //3 seconds to form
 		new /obj/effect/spider/stickyweb(src.loc)
 	else
-		src << "You do not have enough nutrition to create webbing!"
+		src << "You do not have enough nutrition to create webbing! ((You need at least FIVE hundred for a web.))"
```
Comment (SpadesNeil) [inline code/modules/vore/spiderTaurPowers.dm]: Player: "Five hundred what?"

File: code/modules/vore/spiderTaurPowers.dm [hunk 2]
```diff
@@ -21,20 +21,20 @@ obj/item/clothing/suit/web_bindings
 mob/proc/weaveWeb()
 	set name = "Weave Web"
 	set category = "Species Powers"
-	if(nutrition >= 30) //BUGS!
+	if(nutrition >= 500) //People decided to abuse it. Sorry. It was asked to be made so it couldn't be spammed, and what do ya know, people are spamming it everywhere.
```
Comment (SpadesNeil) [inline code/modules/vore/spiderTaurPowers.dm]: This whole proc should be commented out. We shouldn't be able to place webs.

File: code/modules/vore/spiderTaurPowers.dm [hunk 3]
```diff
@@ -21,20 +21,20 @@ obj/item/clothing/suit/web_bindings
...
+		src << "You do not have enough nutrition to create webbing! ((You need at least FIVE hundred for a web.))"
 
 mob/proc/weaveWebBindings()
 	set name = "Weave Web Bindings"
 	set category = "Species Powers"
-	if(nutrition >= 30) //Due to the restrictions of this being a direct proc, I can't put on a time delay.
+	if(nutrition >= 200) //This isn't a huge problem. This is so you can bind people up, and you should only need one of them, two at max.
 		src.visible_message("\blue \the [src] pulls silk from their spinneret and delicately weaves it into bindings.")
-		nutrition -= 30 //So don't abuse this or you'll make the coders cry and raise the nutirition needed.
+		nutrition -= 200 //People abused it and made the coders cry. RIP
```
Comment (SpadesNeil) [inline code/modules/vore/spiderTaurPowers.dm]: Webbing _people_ isn't a problem. The problem is people webbing the rooms. This shouldn't need to be changed very much.

**Comment by SpadesNeil** [comment]
I asked for the area webbing ability to be removed--not nerfed. Binding people was fine as it was.

**Comment by Cameron653** [comment]
Ah, I'll remove it then. The way it was titled was odd, as it said remove it in the title, but said make it not able to be spammed in the description. Changing it now.

**Comment by Cameron653** [comment]
Alright, changed. Commented out the entire laying web proc, and left the web binding suit proc.

**Pull request merged**